src/lib/ppx_coda/versioned.ml

```diff
@@ -256,7 +278,12 @@ let rec generate_core_type_version_decls type_name core_type =
     | Ldot (prefix, "t") ->
         (* type t = A.B.t
            if prefix not whitelisted, generate: let _ = A.B.__versioned__
+           disallow Stable.Latest.t
         *)
+        if is_stable_latest prefix then
+          Ppx_deriving.raise_errorf ~loc:core_type.ptyp_loc
+            "Cannot use type of the form Stable.Latest.t, which may change \
+             over time, within a versioned type" ;
```

Comment:
Nit: I think this comma may be unnecessary.

Reply:
you mean, omit the entire clause within commas?